.github/CODEOWNERS

```diff
@@ -6,4 +6,5 @@
 /src/extensions/resource-deployment/ @ranasaria
 /src/extensions/arc/ @ranasaria
 /src/extensions/azdata/ @ranasaria
-
+/extensions/dacpac/ @kisantia
```

Comment:
is this the right path to use? I see Arvind has /src/ at the start of the paths he added, but the extensions aren't in the src folder

Reply:
yours looks correct, but I would always test it before sending the PR. @ranasaria FYI.

Reply:
you can test by creating a child branch from this one and then ask someone else to create a PR modifying the files in the folder you care about and see whether you are tagged automatically

Reply:
@Benjn helped confirm that this worked :)